src/scenic/core/object_types.py

```diff
@@ -1199,12 +1212,11 @@ def show3D(self, viewer, highlight=False):
 
         viewer.add_geometry(object_mesh)
 
-        # If the camera is not a sphere, render the visible pyramid as a blue wireframe
-        if self.viewAngles != (math.tau, math.pi) or self.visibleDistance != 50:
+        if self.showVisibleRegion:
             camera_pyramid_mesh = self.visibleRegion.mesh.copy()
```

Comment:
If this copy is necessary, could you please add a comment explaining why? It's unintuitive to me.